### DISCUSSION TITLE
Example with Nvidia Nemo

### DISCUSSION THREAD

**Discussion by yogso** [NVIDIA/NVFlare#216] [Q&A] [answered]
Hello,

I am starting to use the framework and I find it very interesting, but I have a doubt: in the documentation you mention that the framework can be used with Nvidia Nemo, do you have any example of an implementation?

Thank you very much.

**Comment by yanchengnv** [comment, marked as answer]
Not yet, but we are planing to work with the Nemo team to develop such examples.

The reason we think NVFLARE could be used for Nemo (and many more) is because the design is generic and not specific to any learning technology. In the future, we plan to develop more examples for different common use cases.

**Reply by YuanTingHsieh** [reply]
can already be found here: https://github.com/NVIDIA/NVFlare/tree/main/integration/nemo
